### PR TITLE
Bugfix/position form state issue

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  // Always include the first and last labels
+  indices.add(0);
+  indices.add(totalLabels - 1);
+
+  // Begin by adding the middle of the entire array
+  addIndices(0, totalLabels - 1);
+
+  // Convert to array and sort to ensure the correct order
+  return Array.from(indices).sort((a, b) => b - a);
+};
```
Comment (jinoosss) [inline packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx:48]: Can you tell me the purpose of this function?
If you want to get equally spaced values, it seems like a good idea to check out the function.

The result of the function behavior.
```
INPUT : calculateMiddleIndices(20,6)
OUTPUT: [19,14,9,6,4,2,1,0]
```

Would the expression below produce a different result? 
```
const spacing = (totalLabels - 1) / (countXAxis - 1);
Array.from({length: countXAxis}, (_, index) => Math.floor(spacing * index))
```

File: packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx [hunk 2]
```diff
@@ -86,7 +91,17 @@ const EarnAddLiquidityContainer: React.FC = () => {
...
+    tokenA, 
+    tokenB, 
+    feeTier: swapFeeTier, 
+    isCreate: createOption?.isCreate, 
+    startPrice: createOption?.startPrice,
+    // defaultPriceRange: [ 
+    //   tickLower ? tickToPrice(Number(tickLower)) : null, 
+    //   tickUpper ? tickToPrice(Number(tickUpper)) : null
+    // ],
+  });
```
Comment (jinoosss) [inline packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx:104]: Remove unused codes.

File: packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx [hunk 3]
```diff
@@ -111,6 +126,12 @@ const EarnAddLiquidityContainer: React.FC = () => {
     addLiquidity,
   });
 
+  // useEffect(() => {
+  //   if(!initialized && router.isReady) {
+     
+  //   }
+  // }, [initialized, router.isReady, router.query]);
```
Comment (jinoosss) [inline packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx:133]: Remove unused codes.

File: packages/web/src/repositories/dashboard/res.ts [hunk 4]
```diff
@@ -0,0 +1,1451 @@
+export const res = {
```
Comment (jinoosss) [inline packages/web/src/repositories/dashboard/res.ts:1]: Remove this file.
If necessary, you can create a mock file.

**Review by jinoosss — COMMENTED** [review]
Great work. 
I left a few comments.

File: packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx:48]: I inherited it from old code so please let me check again

File: packages/web/src/components/pool/exchange-rate-graph/ExchangeRateGraph.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  tokenA,
+  tokenB,
+  feeTier,
+  onSwap,
+  data,
+}: ExchangeRateGraphProps) {
+  return (<ExchangeRateGraphWrapper>
+    <ExchangeRateGraphHeaderWrapper>
+      <p className="title">Exchange Rate</p>
+      {true && (
```
[the remark this answers is not in the document]
Comment (phucvarmeta) [reply packages/web/src/components/pool/exchange-rate-graph/ExchangeRateGraph.tsx:26]: it's a result of my debugging , i will remove it

File: packages/web/src/components/common/line-graph/LineGraph.tsx [hunk 6]
```diff
@@ -118,7 +125,7 @@ function parseTimeTVL(time: string) {
   };
 }
 
-const LineGraph: React.FC<LineGraphProps> = ({
+const LineGraph = ({
```
[the remark this answers is not in the document]
Comment (phucvarmeta) [reply packages/web/src/components/common/line-graph/LineGraph.tsx:128]: I 've set it as another type, i will add the type in

File: packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/components/pool/exchange-rate-graph-content/ExchangeRateGraphContent.tsx:48]: For this case i re-used the earlier code, i tried the new approach and it worked, thanks.

**Review by jinoosss — APPROVED** [review]
Great work.
Thank you for adding comments to code.

**Pull request merged**